src/Bundle/DependencyInjection/SyliusResourceExtension.php

```diff
@@ -320,4 +324,43 @@ private function loadResources(array $loadedResources, ContainerBuilder $contain
             }
         }
     }
+
+    private function registerMetadataConfiguration(ContainerBuilder $container, array $config): void
+    {
+        $resources = $this->getResourcesToWatch($container, $config);
+
+        $container->getDefinition('sylius.metadata.resource_extractor.php_file')->replaceArgument(0, $resources);
+    }
+
+    private function getResourcesToWatch(ContainerBuilder $container, array $config): array
+    {
+        $resources = [];
```

Comment:
It's doesn't work with resources, but rather resource php configuration files.

```suggestion
    private function getResourceFilesToWatch(ContainerBuilder $container, array $config): array
    {
        $resourceFiles = [];
```

Reply:
done, you are right.